CMSgov/easi-app
# [EASI-2439] - System profile edit team

# EASI-2439

[Figma](https://www.figma.com/file/RhVQgXxE6tyj33TygymuhD/System-List-and-Profile-(Hi-fi)?node-id=2865%3A80969)

## Changes and Description

- Routes for edit team page and form
  - Edit team page: `/systems/:systemId/team/edit`
  - Add/edit team member form: `/systems/:systemId/team/edit/team-member`
- Edit button/box to direct to edit form  
  - `<EditPageCallout>` (src/views/SystemProfile/components/EditPageCallout/index.tsx)
  - Commented out until work in EASI-2447 is done
- Edit team page (src/views/SystemProfile/components/Team/Edit/index.tsx)
  - Contact cards for each team member
- Link to edit role and remove contact
  - Functionality will be implemented in EASI-2447
- Add contact button
  - Links to add/edit team member form that will be built out in EASI-2447
- Edit number of employees and contractors fields
  - We don't currently have the ability to edit this data in CEDAR. These fields were created but commented out until the backend work can be completed (confirmed this with Clay, Zoe, and Jeff).



## How to test this change

1. Connect to VPN
2. Remove comment block around `<EditPageCallout>` in `src/views/SystemProfile/index.tsx`
3. Visit [/systems/408-4104-0/team](http://localhost:3000/systems/408-4104-0/team)
4. Confirm Edit Team callout is in left sidebar
5. Click "Edit Team" button (or go to [/systems/408-4104-0/team](http://localhost:3000/systems/408-4104-0/team))
6. Confirm Edit Team page matches Figma




## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
